packages/node_modules/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -1047,17 +1047,26 @@ test("Case-sensitive tag has capitalization preserved", function() {
 //          "circle tag inside block inside svg has an svg namespace" );
 // });
 
-// QUnit.skip("Block helper with root foreignObject allows namespace to bleed through", function() {
-//   registerYieldingHelper('testing');
-
-//   let template = compile('<foreignObject>{{#testing}}<div></div>{{/testing}}</foreignObject>');
-
-//   let fragment = render(template, { isTrue: true }, env, { contextualElement: document.createElementNS(svgNamespace, 'svg') }).fragment;
-//   let svgNode = fragment.firstChild;
-//   equal( svgNode.namespaceURI, svgNamespace,
-//          "foreignObject tag has an svg namespace" );
-//   equal( svgNode.childNodes[0].namespaceURI, xhtmlNamespace,
-//          "div inside morph and foreignObject has xhtml namespace" );
-// });
+test("HTML namespace from root element is continued to child templates" function() {
+  compilesTo(
+    '<svg>{{#unless}}<circle />{{/unless}}</svg>',
+    "<svg><circle /></svg>"
+  );
+  let svg = root.firstChild;
+  let circle = svg.firstChild;
+  equal(svg.namespaceURI, SVG_NAMESPACE);
+  equal(circle.namespaceURI, SVG_NAMESPACE);
+});
 
+test("root <foreignObject> tag is SVG namespaced" function() {
```

Comment:
Can we add these tests in the updating test suite instead of adding it here? Going forward, we'd like all new tests to test both initial render and updating :smile: